goaviatrix/gateway.go

```diff
@@ -1130,3 +1132,27 @@ func (c *Client) DisableEgressTransitFirenet(transitGateway *TransitVpc) error {
 	}
 	return c.PostAPI(action, data, BasicCheck)
 }
+
+func (c *Client) EnableMonitorGatewaySubnets(gateway *Gateway) error {
+	action := "enable_monitor_gateway_subnets"
+	MonitorInstanceList := gateway.MonitorExcludeList
```

Comment:
I don't think the extra variable `MonitorInstanceList` is necessary, just use `gateway.MonitorExcludeList` directly.